src/vl-map-layer.js

```diff
@@ -175,22 +190,35 @@ export class VlMapLayer extends VlElement(HTMLElement) {
         }
     }
 
-    _auto_extentChangedCallback(oldValue, newValue) {
-        this.__zoomToExtent();
+    /**
+     * Zoom naar alle features in deze layer.
+     *
+     * @param {number} maxZoom - Hoe diep er maximaal ingezoomd mag worden.
+     * @returns {Promise<void>}
+     */
+    async zoomToExtent(maxZoom) {
+        await this._mapReady;
+        if (this._map) {
+            this._map.zoomToExtent(this.__boundingBox, maxZoom);
+        }
```

Comment:
idem als hierboven. ik zou hier geen "null" check verwachten en een error throwen als de map er niet is?

Reply:
ook hier zou ik durven zeggen dat de if-test gewoon overbodig is nu dat we wachten op de ready

Reply:
Ik doe de test gewoon weg aangezien die by default true gaat geven als de map ready is.